ovn/ovn-nb.ovsschema

```diff
@@ -269,6 +274,16 @@
                     "type": {"key": "string", "value": "string",
                              "min": 0, "max": "unlimited"}}},
             "isRoot": false},
+        "Logical_Router_Policy": {
```

Comment:
Do we want to attach a "name" to policy as well?

Reply:
Done.